extensions/security/iprange/pom.xml

```diff
@@ -64,5 +64,28 @@
         </dependency>
 
     </dependencies>
+
+    <build>
+    <plugins>
+        <plugin>
+            <groupId>eu.somatik.serviceloader-maven-plugin</groupId>
```

Comment:
Can you tell me about this plugin and its purpose?

Reply:
it scans code and generates service loader config files in the META-INF directory.

<img width="559" alt="image" src="https://user-images.githubusercontent.com/1700062/102122742-3c959180-3e46-11eb-8266-8d53d26c2a48.png">

Reply:
I think the Maven Plugin is unnecessary though and for the feature it provides it will slow down the build. I think it would be better just to add the file `META_INF/services/org.exist.http.servlets.ExistExtensionServlet` to the `src/resources` folder of the module manually.